po/fr.po

```diff
@@ -19,14 +20,15 @@ msgstr ""
 "Project-Id-Version: fr\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2018-01-13 20:39+0100\n"
-"PO-Revision-Date: 2014-01-18 16:04-0600\n"
-"Last-Translator: Sébastien Villemot <sebastien@debian.org>\n"
-"Language-Team: \n"
+"PO-Revision-Date: 2018-10-25 13:10+0200\n"
+"Last-Translator: La Boussole <yoann@laboussole.coop>\n"
+"Language-Team: français <yoann@laboussole.coop>\n"
 "Language: French\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=2; plural=n>1\n"
+"Plural-Forms: \n"
+"X-Generator: Gtranslator 2.91.7\n"
```

Comment:
You should configure the Plural-Forms in your Gtranslator or it has a bug.
For now I have fixed it.